app/src/main/java/com/example/whatshouldiread/ui/bookshelf/BookshelfFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.whatshouldiread.ui.bookshelf
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.example.whatshouldiread.R
+
+class BookshelfFragment : Fragment() {
+
+    private lateinit var bookshelfViewModel: BookshelfViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        bookshelfViewModel =
+            ViewModelProviders.of(this).get(BookshelfViewModel::class.java)
+        val root = inflater.inflate(R.layout.fragment_bookshelf, container, false)
```

Comment:
`val root = inflater.inflate(R.layout.fragment_bookshelf, container, false)`
This references an XML that currently doesn't exist

Reply:
Fixed in commit 9dc7836657537c2eb8639a1b7d8326cb16d46580
Added in XML so compile completes